src/components/alerts/alert.jsx

```diff
@@ -53,6 +57,30 @@ const AlertComponent = ({
                 />
             ) : content}
         </div>
+        {showSaveNow && (
+            <button
+                className={styles.alertConnectionButton}
+                onClick={onSaveNow}
+            >
+                <FormattedMessage
+                    defaultMessage="Try&nbsp;Again" // TODO control wrapping in css
```

Comment:
Maybe we shouldn't localize this yet if we know it's going to change, don't want to unnecessarily ask people to re-translate (or try to translate strings with HTML entities and possibly break things when they do).